tags/TagMapContext.md

```diff
@@ -0,0 +1,144 @@
+# Summary
+A `Tag` is a key-value pair that can be used to label anything that is associated
+with a specific operation, such as an HTTP request. These `Tag`s are used to
+aggregate measurements in a [`View`](https://github.com/census-instrumentation/opencensus-specs/blob/master/stats/DataAggregation.md#view) 
+according to unique value of the `Tag`s. The `Tag`s can also be used to filter (include/exclude)
+measurements in a `View`.
+
+# TagMap 
```

Comment:
Mention that this is the recommended name but languages can have a more language specific name like in go "tag.Map" because how the packages work in go.